nvflare/app_opt/xgboost/histogram_based_v2/executor.py

```diff
@@ -85,10 +85,16 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
             adaptor.set_abort_signal(self.abort_signal)
             adaptor.initialize(fl_ctx)
             self.adaptor = adaptor
+        elif event_type == Constant.EVENT_XGB_ABORTED:
+            error = fl_ctx.get_prop(FLContextKey.FATAL_SYSTEM_ERROR)
+            self.log_error(fl_ctx, error)
+            self.system_panic(reason=error, fl_ctx=fl_ctx)
```

Comment:
Please check with YT about PR https://github.com/NVIDIA/NVFlare/pull/2674. There he changed _notify_client_done() to call system_panic() if rc != 0. I think we need to keep that change, then you don't need to call system_panic() here since the rc Constant.EXIT_CODE_JOB_ABORT is not 0.